config.py

```
@@ -0,0 +1,418 @@
+    # Configure root logger
+    logging.basicConfig(
+        level=log_level,
+        handlers=handlers,
+        force=True
+    )
+    
+    # Set correlation ID if enabled
+    if config.logging.correlation_id:
+        logging.correlation_id = None  # Will be set per request
+    
+    logger.info(f"Logging configured with level={config.logging.level}, format={config.logging.format}")
```

Ruff (0.12.2)

Logging statement uses f-string

(G004)